metadata-service/plugin/src/main/java/com/datahub/plugins/common/PluginType.java

```diff
@@ -0,0 +1,13 @@
+package com.datahub.plugins.common;
+
+/**
+ * Supported plugin types
+ */
+public enum PluginType {
+  AUTHENTICATOR, AUTHORIZER;
```

Comment:
Let's add javadocs to each type in the enum

Reply:
please respond&

Reply:
Added